jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/jobdetails/JobDetailsViewImpl.html

```diff
@@ -0,0 +1,41 @@
+<div data-field="container" class="container-fluid">
+    <!-- tabs -->
+    <div>
+        <!-- navigation tabs -->
+        <ul class="nav nav-tabs nav-tabs-pf" id="navTabs">
+            <li class="active"><a href="#" data-i18n-key="Job_Details"></a></li>
+        </ul>
+    </div>
+
+    <!-- Page content -->
+    <div>
+        <div data-field="job-details"></div>
+        <div class="row">
+            <div class="col-md-12">
+                <div class="form-horizontal">
+                    <div class="form-group" data-field="errorControlGroup">
+                        <label class="control-label col-md-4" data-i18n-key="Exceptions_Occurred"></label>
+                        <div class="col-md-8">
+                            <textarea class="col-sm-12"
+                                      rows="10" readonly data-field="job-errors"></textarea>
+                        </div>
+                    </div>
+                    <div class="form-group hidden" data-field="params-form-group">
```

Comment:
@cristianonicolai I don't understand the error message you have, the data-field is here. Tested in my local environment it works.